src/components/resources-section/resource-labels/ResourceLabels.js

```diff
@@ -0,0 +1,31 @@
+import { Button } from 'primereact/button';
+import icons from '@utils/icons';
+import styles from './ResourceLabels.module.scss';
+
+const DisplayLabel = ({ label }) => (
```

Comment:
Why not taking this component to a different file?

Reply:
Will do on refactor branch